projectforum/static/js/project-list.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function() {
+
+    // JQuery code to be added in here.
+    $('.search').click(function() {
+        var url = window.location.href;
+        var filters = $('.keywords').val();
+        var sorting = parseInt($('.sort-projects').val());
+        var queryloc = url.indexOf('?');
+        var ascending;
+        var salary = '';
```

Comment:
If you are going to put all your vars at the top, you could separate them by commas.